Content.Shared/GameObjects/Components/Explosion/SharedClusterFlashComponent.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Components.UserInterface;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.GameObjects.Components.Explosion
+{
+    [Serializable, NetSerializable]
+    public enum ClusterFlashVisuals : byte
+    {
+        GrenadesCounter,
+        GrenadesMax,
```

Comment:
not used anymore

Reply:
GrenadesCounteer is used, so i removing only GrenadesMax